locale/de_DE.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -213,11 +225,11 @@ msgstr "Linux-Benutzer können ihr X Display wie folgt teilen:"
 
 #: index.php:1218
 msgid "Share desktop (free UltraVNC / Windows only)"
-msgstr "Desktopfreigabe (UltraVNC (freie Software) / Windows)"
+msgstr "Desktopfreigabe (Windows)"
 
 #: index.php:1225
 msgid "Share desktop (non free RealVNC)"
-msgstr "Desktopfreigabe (RealVNC (kommerziell) / OS X)"
+msgstr "Desktopfreigabe (Mac)"
```

Comment:
"macOS" would be fine. What do you think about removing the application name in the msgid and use operating system names instead?